linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/oval/shared.xml

```diff
@@ -32,6 +44,23 @@
     <ind:instance datatype="int">1</ind:instance>
   </ind:textfilecontent54_object>
 
+{{% if 'sle' in product %}}
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+    comment="Tests that
+    /usr/lib/systemd/systemd-sulogin-shell
+    was not removed from the default systemd emergency.service to ensure that a
+  password must be entered to access single user mode"
```

Comment:
There are some minor indentation issues here.

Reply:
Thanks :bow: with [d1fe359](https://github.com/ComplianceAsCode/content/pull/10482/commits/d1fe359bba6deebd0482096bcfafa4ee1849f591) should be better